tests/build.gradle.kts

```diff
@@ -30,11 +30,12 @@ import io.spine.internal.dependency.JUnit
 
 buildscript {
 
-    val baseRoot = "$rootDir/.."
-    val versionGradle = "$baseRoot/version.gradle.kts"
+    val baseRoot = "${rootDir}/.."
+    val versionGradle = "${baseRoot}/version.gradle.kts"
+    val commonPath = io.spine.internal.gradle.Scripts.commonPath
```

Comment:
In this context, I'd rename it to `sriptsPath` as it's not clear for whom it is common.